scss/os/_vectorstylecontrol.scss

```diff
@@ -0,0 +1,5 @@
+.c-vectorstylecontrol__borderdash {
+  fill: none;
+  stroke: $gray-500;
```

Comment:
I would suggest using `$input-color` here, to match the font color used for other form inputs.